kamailio/presence-fast-pickup.cfg

```diff
@@ -60,11 +60,11 @@ route[PRESENCE_FAST_PICKUP_ATTEMPT]
            route(PRESENCE_FAST_PICKUP_OPTION);
            $du = $sht(fp=>$var(md5));
            append_hf("Replaces: $var(replaced_call_id)$var(Pickup)\r\n");
+           append_hf("k-cid: $var(replaced_call_id)\r\n");
```

Comment:
@lazedo the "Replaces: " header already has the replaced_call_id, should use that to avoid UDP fragmentation and/or duplication of info

Reply:
this is between kamailio and fs and serves only for packet capture correlation. we don't want to include X-* as it will end up in kazoo which is not intended